src/tv2-common/content/server.ts

```diff
@@ -28,6 +28,7 @@ export interface MakeContentServerSourceLayers {
 	Sisyfos: {
 		ClipPending: string
 		StudioMicsGroup: string
+		SisyfosPersistedLevels: string
```

Comment:
Just a FYI: In the refactor for "acceptsPersist" and "wantsToPersist" on develop we introduced "SisyfosPersistMetaData" which is where we store which layers that should be persisted.

We just need to be aware of, that this should utilize that, when this is merged with develop

Reply:
Yup, I'll make a separate PR for develop with adequate changes so that we don't have to resolve it later during automerge :)